docs/source/changelog.rst

```diff
@@ -1,6 +1,18 @@
 Changelog
 =========
 
+2021.08.0
```

Comment:
This will be 2021.08.1, because s3fs has a release in between due to aiobotocore; I'd like the versions to match exactly.